schema/deploy/tables/ciip_user.sql

```diff
@@ -33,6 +33,53 @@ create unique index user_email_address_uindex
   on ggircs_portal.ciip_user(uuid);
 
 
+do
+$grant$
+begin
```

Comment:
I'm unsure of the nested transaction:
> If DO is executed in a transaction block, then the procedure code cannot execute transaction control statements. Transaction control statements are only allowed if DO is executed in its own transaction.
https://www.postgresql.org/docs/11/sql-do.html

May be necessary?

Reply:
Can we test to ensure that if something fails in the `perform` that the `sqitch` run is aborted?

Reply:
Alternatively, if the `do` blocks are here to squelch useless output, perhaps the return value of the `grant_permissions` function could be improved so a `select` would produce useful output in the `sqitch` log.

Reply:
A failed `perform` inside the nested `do` block does abort the sqitch run and triggers a revert